docs/samples/Microsoft.ML.Samples/Dynamic/ValueMapping.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.Data.DataView;
+using Microsoft.ML.Data;
+using Microsoft.ML.Transforms.Conversions;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public class ValueMappingExample
+    {
+        class SampleInfertDataWithFeatures
+        {
+            public float Age = 0;
+            public string Education = default;
+            public string EducationCategory = default;
+        }
+
+        ///<summary>
```

Comment:
This sample is missing the "So what?" aspect. Why would I use this transform, and where is it helpful?

Can you give an example of a problem you can't solve without using this?